iati/core/tests/test_default.py

```diff
@@ -58,6 +58,19 @@ def test_default_schemas(self):
         schemas = iati.core.default.schemas()
 
         assert isinstance(schemas, dict)
-        assert len(schemas) == 1
+        assert len(schemas) == 2
         for _, schema in schemas.items():
-            assert isinstance(schema, iati.core.Schema)
+            assert isinstance(schema, iati.core.ActivitySchema) or isinstance(schema, iati.core.OrganisationSchema)
+
+    @pytest.mark.parametrize("invalid_name", iati.core.tests.utilities.find_parameter_by_type(['str', 'none', 'view', 'set', 'mapping', 'list', 'bytearray', 'memory', 'range'], False))
```

Comment:
This leaves a somewhat strange list of types that are checked - a mixture of things that are kinda-strings, and things that are not.

If there is a particular reason for the list of types being as it is, the reason should be documented.